### PR TITLE
Add `Relaxed Query Expression Keywords` section to SLB3 release note

### PR DESCRIPTION
**Pull request by rdulmina**
## Purpose
$title

### CODE REVIEW

File: release-notes/swan-lake-beta3-release-note.md [hunk 1]
```diff
@@ -271,6 +271,17 @@ if x == () {
 }
 ``` 
 
+##### Relaxed Query Expression Keywords
+
+Keywords, that are used in query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, `select`) can now be used as identifiers outside the query context.
```
Comment (praneesha) [inline release-notes/swan-lake-beta3-release-note.md:276]:
```suggestion
Keywords that are used in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, `select`) can now be used as identifiers outside the query context.
```

**Review by praneesha — APPROVED** [review]

File: release-notes/swan-lake-beta3-release-note.md (hunk 1 above)
Comment (MaryamZi) [reply release-notes/swan-lake-beta3-release-note.md:276]: How about something like

```suggestion
Identifiers that are keywords in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, and `select`) can now be used as ordinary identifiers outside query contexts. i.e., they are no longer required to be quoted identifiers.
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 2]
```diff
@@ -271,6 +271,17 @@ if x == () {
...
+
+```ballerina
+// Now allowed.
+int limit = 5;
+// `string:'join()` now can be used without initial quote.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta3-release-note.md:281]:
```suggestion
// `string:'join()` is no longer required to use quoted identifiers.
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 3]
```diff
@@ -271,6 +271,16 @@ if x == () {
 }
 ``` 
 
+##### Relaxed Query Expression Keywords
+
+Identifiers that are keywords in a query expression context (`where`, `join`, `order`, `by`, `equals`, `ascending`, `descending`, `limit`, `outer`, and `select`) can now be used as ordinary identifiers outside query contexts. i.e., they are no longer required to be quoted identifiers.
+```ballerina
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta3-release-note.md:277]:
```suggestion

```ballerina
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 4]
```diff
@@ -271,6 +271,16 @@ if x == () {
...
+// Now allowed.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta3-release-note.md:278]:
```suggestion
// Now allowed. No longer required to use a quoted identifier (`'limit`).
```

File: release-notes/swan-lake-beta3-release-note.md [hunk 5]
```diff
@@ -271,6 +271,16 @@ if x == () {
...
+int limit = 5;
+// `string:'join()` is no longer required to use quoted identifiers.
```
Comment (MaryamZi) [inline release-notes/swan-lake-beta3-release-note.md:280]:
```suggestion
// Now allowed.  No longer required to use a quoted identifier (`string:'join()`).
```

**Review by MaryamZi — APPROVED** [review]

**Review by praneesha — APPROVED** [review]

**Pull request merged**